molo/commenting/templates/comments/comment.html

```diff
@@ -5,7 +5,7 @@ <h4 class="comment__author">
       {{node.user_name}}
     {% else %}
       {% if not node.user.profile.alias %}
-        {% trans "Anonymous" %}
+        {% trans settings.commenting.CommentingSettings.commenting_anonymous%}
```

Comment:
last thing we can test for is that if the default is in another language, that it shows the translation for that. ie if we display the default alias in a spanish (or other language) site, that it shows the spanish version for anonymous?

Reply:
@codiebeulaine @sewagodimo I don't think we are able to translate a value of variable. For Pootle you would need the text that you want to be translated in the template. so the above syntax is not right.